readme.md

```diff
@@ -347,13 +347,15 @@ export default class extends React.Component {
     super(props);
     this.state = { points: local.points };
     this.updatePoints = this.updatePoints.bind(this);
-    subscribe(local, 'points', this.updatePoints);
   }
   updatePoints (points) {
     this.setState({ points });
   }
+  componentDidMount () {
+    this.subscription = subscribe(local, 'points', this.updatePoints);
+  }
   componentWillUnmount () {
-    unsubscribe(this.updatePoints);
+    unsubscribe(this.subscription);
```

Comment:
This is an example of unsub with the cb: "_You can also unsubscribe by the callback_", so the previous commit was correct